apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -233,6 +234,8 @@ public void loadSampleU() {
                     lastUpdated.after(Date.from(Instant.now().minus(1, ChronoUnit.MINUTES))));
               });
 
+      assertEnrollments(beneficiaryFromDb);
```

Comment:
As mentioned above, I think you'll want to have several test cases here, covering the following scenarios:
1. If we insert a bene, do the resulting DB records have only the correct months' data populated, or does it end up with a lot of weird null-ish entries?
2. If we update a bene with the current year's data and just more months filled in, does the resulting `Enrollments` data look as expected?
3. If we update a bene with the current year's data and *less* months filled in (shouldn't happen but the schema allows it so we should ahndle it just in case), do the resulting DB records look as expected?
4. If we update a bene with a new year's data, is it appended to the previous years' data, or does it overwrite it?